modmail/database/models/emojis.py

```diff
@@ -0,0 +1,16 @@
+from tortoise import fields
+from tortoise.models import Model
+
+from .messages import Messages
+
+
+class Emojis(Model):
```

Comment:
possible guild field?

Reply:
Emoji are mapped to messages, which contain the `guild_id` so there isn't a need to add guild field to this.